client/packages/system/src/Manage/Preferences/EditPage/PreferenceAccordion.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  ChevronDownIcon,
+} from '@openmsupply-client/common';
+import React, { ReactNode } from 'react';
+
+interface PreferenceAccordionProps {
+  label: string;
+  children: ReactNode;
+  sx?: Record<string, unknown>;
+}
+
+export const PreferenceAccordion = ({
+  label,
+  children,
+  sx,
+}: PreferenceAccordionProps) => {
+  return (
+    <Accordion
+      sx={{
+        marginTop: 1,
+        border: '1px solid',
+        borderColor: 'grey.400',
+        borderRadius: 1,
+        boxShadow: 'none',
+        ...sx,
```

Comment:
To fix the inconsistent top border, you can add:
```
    '&::before': {
          display: 'none', // Common use case - hide the default border
        },
```

Changes it from this:
<img width="656" height="129" alt="Image" src="https://github.com/user-attachments/assets/95ff641b-ffa8-4625-a988-1ed1072f5b2b" />
to this:

<img width="622" height="124" alt="Image" src="https://github.com/user-attachments/assets/d92a2e07-a27c-4e17-91c8-f9e2d04977f0" />